src/index.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import { createStore } from 'redux';
+import { createStore, combineReducers } from 'redux';
 import 'normalize-css/normalize.css';
 
 import './index.scss';
 import { App } from './App';
 import * as serviceWorker from './serviceWorker';
-import { reducer } from './store/reducers/init-game';
+import { initGameReducer } from './store/reducers/init-game';
+import { submitAnswerReducer } from './store/reducers/submit-answer';
+
+const rootReducer = combineReducers({
+    initGame: initGameReducer,
+    submitAnswer: submitAnswerReducer,
```

Comment:
this is incorrect

keys of an object you pass to `combineReducers` correspond to state properties

state is a description of current state/shape/truth of the system, you should think about it as something static, in terms of nouns and not in terms of verbs

for example, the widely used example of the simplest app you can imagine is a button which you can click and click count is displayed somewhere, in this instance state would have property `count` (and not `increment` or `increaseCount`); similarly, typical state properties would be `subject`, `user`, `author`, `post`, `superImportantThing`, `something` (these are all nouns)

here you use `initGame`, `submitAnswer` which are verbs, so they correspond to a change/action, something dynamic

so it goes this way:
redux action -> verb, e.g. `doSomething`, `increment`
state property -> noun, e.g. `something`, `count`